torch_xla/csrc/ops/dynamic_ir.cpp

```diff
@@ -41,6 +41,9 @@ int64_t SizeNode::getDynamicValue() const {
   }
   torch::lazy::NodePtr cloned =
       torch::lazy::MakeNode<SizeNode>(operands_[0], dim_);
+  // Wrap the IR of SizeNode into a dummy tensor and execute/fetch the value
+  // of this tensor. GetTensors will return a cpu at::Tensor so we can just
+  // extract the value of it.
   std::vector<XLATensorPtr> dummy_size_tensors = {
```

Comment:
It seems we're using 
```
XLATensorPtr XLATensor::Create(
    torch::lazy::Value ir_value, const torch::lazy::BackendDevice& device,
    c10::optional<at::ScalarType> logical_element_type)
```
but `clone` is a torch::lazy::NodePtr object. How is the `torch::lazy::NodePtr` converted to a `torch::lazy:Value`? Is it by using torch::lazy:Value's implicit constructor
```
Value(NodePtr&& node, size_t index = 0)
      : node(std::move(node)), index(index) {}
```
?

Reply:
Never mind, I think so. It uses the conversion constructor.